scipy/stats/_kde.py

```diff
@@ -574,17 +577,25 @@ def _compute_covariance(self):
         covariance_factor().
         """
         self.factor = self.covariance_factor()
```

Comment:
@steppi how about this?
```suggestion
        """
        # To evaluate the PDF, we need to compute a product 
        # (x - µ)^T P (x - µ), where x are the points at which the PDF is
        # to be evaluated, µ are the means of the Gaussians (the points in)
        # the dataset) and P is the precision matrix (the inverse of the
        # covariance matrix of the kernel).
        # The naive implementation would be to compute the difference 
        # d = x - µ first, then compute d_ij^T P d_ij  for every pair of
        # i and j. A speedup was obtained by performing the Cholesky
        # decomposition P = L L^T, so that (x - µ)^T P (x - µ) becomes
        # ((x - µ)^T L ) ((x - µ)^T L )^T, which can be computed by first
        # "whitening" x and µ separately (i.e. x^T L and µ^T L), then computing
        # pairwise differences, and finally taking a sum of squares.
        # Now, we want to eliminate explicit computation of the precision 
        # matrix P to avoid associated numerical and performance disadvantages.
        # It turns out that we can compute products like x^T L without first 
        # inverting P then computing Cholesky factors L. Instead, we reverse
        # rows and columns of the covariance matrix, compute the Cholesky 
        # factor of the permuted matrix, and reverse the rows and columns of
        # the Cholesky factor. Performing a triangular solve with this factor
        # can be used to calculate x^T L. See:
        # https://mathoverflow.net/questions/230808/computing-the-inverse-of-a-cholesky-decomposition/230817#230817  # noqa
        # and gh-16692 for details.
        self.factor = self.covariance_factor()
```

Reply:
Looks good!